xbmc/network/NetworkServices.cpp

```diff
@@ -491,7 +497,9 @@ void CNetworkServices::Start()
   StartZeroconf();
 #ifdef HAS_WEB_SERVER
   if (m_settings->GetBool(CSettings::SETTING_SERVICES_WEBSERVER) && !StartWebserver())
+  {
```

Comment:
I'll remove it (it's not wrong style though)

Reply:
IIRC our coding style guide says that we don't use curly braces on single line blocks (even though I think it's a bad style).

Reply:
No, it's not explicit about that particular point, because there was some debate. In fact, the example show both possibilities (with and without braces). I know it because I rewrote most of it  last year or so ;-)